src/scripts/migrateProjects.js

```diff
@@ -0,0 +1,520 @@
+/**
+ * name : migrateProjects.js
+ * author : Priyanka Pradeep
+ * created-date : 24-Aug-2024
+ * Description : script to create the project from consumption side.
+ */
+
+require('module-alias/register')
+require('dotenv').config({ path: '../.env' })
+require('../configs/events')()
+const fs = require('fs')
+const path = require('path')
+const createCsvWriter = require('csv-writer').createObjectCsvWriter
+const entityTypeService = require('@services/entity-types')
+const projectService = require('@services/projects')
+const entityService = require('@services/entities')
+const resourceService = require('@services/resource')
+const resourceQueries = require('@database/queries/resources')
+const _ = require('lodash')
+const MongoClient = require('mongodb').MongoClient
+// var ObjectId = require('mongodb').ObjectID
+
+//get mongo db url
+const mongoUrl = process.env.MONGODB_URL
+
+if (!mongoUrl) {
+	throw new Error('MONGODB_URL is not set in the environment variables.')
+}
+
+// Path to the CSV file
+const outputPath = path.resolve(__dirname, 'migration_results.csv')
```

Comment:
will the csv be able to handle if the result is volume is more? @priyanka-TL  
Should we go for a better approach?